content/TernaryOperator/TernaryOperator.facts.md

```diff
@@ -0,0 +1,12 @@
+### Facts for Ternary Operator lesson:
+
+The ternary operator `?:` is a conditional operator that simplifies an `if/else` statement. It has three operands: a condition, a value for `true`, and a value for `false`.
```

Comment:
Sorry, but I have tested this content several times and it does not detect any plagiarism issues. Please exclude the title part since that will not appear in the reader. Please check this one. Thank you.
![image](https://user-images.githubusercontent.com/25945790/73143732-ad21ab80-40d8-11ea-8a2f-56e0e833c559.png)

Reply:
Noted. Thanks.